recipes/WHAMandWHAMR/enhancement/train.py

```diff
@@ -140,7 +141,9 @@ def compute_objectives(self, predictions, targets):
         """Computes the si-snr loss"""
         predicted_wavs, predicted_specs = predictions
 
-        if self.use_freq_domain:
+        if (
+            self.use_freq_domain and predicted_specs is not None
```

Comment:
same for predicted specs? Can it be None?

Reply:
same.

@ycemsubakan ?